tools/regression_tests/src/web/assets/jquery.js

```diff
@@ -0,0 +1,5 @@
+/*! jQuery v1.11.3 | (c) 2005, 2015 jQuery Foundation, Inc. | jquery.org/license */
```

Comment:
That issue is unavoidable. If you don't update the code it will be using old versions whether they are fetched from the network or from disk.